truffle/com.oracle.truffle.api/src/com/oracle/truffle/api/frame/FrameInstance.java

```diff
@@ -47,9 +48,18 @@
     /** @since 0.8 or earlier */
     boolean isVirtualFrame();
 
-    /** @since 0.8 or earlier */
+    /**
+     * @return the node that is calling the next {@link CallTarget target} or <code>null</code> if
+     *         its the topmost frame or the next {@link CallTarget target} was called without a
+     *         {@link TruffleRuntime#createDirectCallNode(CallTarget)} or
+     *         {@link TruffleRuntime#createIndirectCallNode() indirect} call node.
+     * @since 0.8 or earlier
+     **/
```

Comment:
What does "next" mean in this context? It means "caller"? Or the node representing the "call site" that invoked the method with this frame instance?